src/soloud/src/backend/miniaudio/soloud_miniaudio.cpp

```diff
@@ -144,5 +149,27 @@ namespace SoLoud
         aSoloud->mBackendString = "MiniAudio";
         return 0;
     }
+
+    result miniaudio_changeDevice_impl(void *pPlaybackInfos_id)
```

Comment:
Are we sure we want to add these things straight into `soloud_miniaudio.cpp`? Won't this be overridden when we upgrade SoLoud (leading to possibly gnarly merge conflicts)?

Reply:
I tried to see if there was another place. This is the only place where, those who choose to use miniaudio have access to their APIs.

Reply:
Gotcha, thanks for checking!